test/hotspot/gtest/concurrent_test_runner.inline.hpp

```diff
@@ -92,9 +86,9 @@ class ConcurrentTestRunner {
   }
 
 private:
-  long testDurationMillis;
-  int nrOfThreads;
-  TestRunnable* unitTestRunnable;
+  const TestRunnable* unitTestRunnable;
```

Comment:
you made it a pointer to const TestRunnable, not a const pointer to TestRunnable.
```suggestion
  TestRunnable* const unitTestRunnable;
```